ui/src/criteria/unhintedValue.tsx

```diff
@@ -240,3 +259,60 @@ function UnhintedValueInline(props: UnhintedValueInlineProps) {
     </GridLayout>
   );
 }
+
+function decodeComparisonOperator(
+  operator: dataProto.UnhintedValue_ComparisonOperator
+): tanagraUI.UIComparisonOperator {
+  switch (operator) {
```

Comment:
Any chance there's a `decodeByName` utility function for ts enums? Just curious, I have this same problem on the Java side converting the OpenAPI enums.

Reply:
That can't really work here because the proto enums are integers, otherwise that is possible, though as the data changes that sometimes breaks down.